README.md

```diff
@@ -107,8 +112,12 @@ cd crewlink-server
 ```sh
 yarn install
 ```
-3. Compile and run the project
-```JS
+3. Set the ADDRESS variable to your server URL
```

Comment:
Where do you do this? What file? I am assuming `.env` but not everyone will know about such files

Reply:
In this case I was supposing you'd set it as a shell variable (see the example snippet, though I suppose that code snippet wouldn't work on Windows)... maybe a .env file would be a better idea as it'd persist?